FPS-Submission_Guidelines.md

```diff
@@ -138,8 +138,8 @@ Upon submission of a PR, a series of technical checks will run on GitHub to veri
 		<li>Each domain must satisfy the /.well-known/ metadata requirement:</li>
 		<ul>
 <li>The /.well-known/ metadata requirement demonstrates that the submitter has administrative access to the domains present in the set, since administrative access is required to modify the /.well-known/ file. This will help prevent unauthorized actors from adding domains to a set. </li>
```

Comment:
Should we add a note somewhere about how we'll handle the legacy `.well-known/first-party-set` files going forward?

Reply:
Added a line in the Submission Guideline where it discusses hosting the well-known.